sql/sql_lex.cc

```diff
@@ -41,6 +41,7 @@
 #include "sql_insert.h"                // Sql_cmd_insert_base
 #include "lex_token.h"
 
+#pragma GCC diagnostic ignored "-Wchar-subscripts"
```

Comment:
Don't know who wrote the code, leave them alone, but turn off the alarm at compile time to avoid affecting the mood

Reply:
> Don't know who wrote the code, leave them alone, but turn off the alarm at compile time to avoid affecting the mood

`sql_lex.cc` may be written by monty.